src/main/scala/com/softwaremill/mqperf/mq/PulsarMq.scala

```diff
@@ -0,0 +1,84 @@
+package com.softwaremill.mqperf.mq
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+import com.softwaremill.mqperf.config.TestConfig
+import com.typesafe.scalalogging.StrictLogging
+import org.apache.pulsar.client.api._
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+class PulsarMq(testConfig: TestConfig) extends Mq with StrictLogging {
+
+  private val PulsarSubscriptionId = "mqperf-group"
+  private val Topic = "mqperf"
+
+  private def pulsarHosts = testConfig.brokerHosts.map(_ + ":6650").mkString(",")
+
+  override type MsgId = Message[Array[Byte]]
+
+  override def createSender() =
+    new MqSender {
+
+      val pulsarClient = PulsarClient.builder()
+        .serviceUrl(s"pulsar://$pulsarHosts")
+        .build()
+
+      val producer: Producer[Array[Byte]] = pulsarClient.newProducer
+        .topic(Topic)
+        .create
+
+      /* Ack level is set on Bookkeeper level via CLI or REST or startup parameter
+      https://pulsar.apache.org/docs/en/administration-zk-bk/
+      Set persistence policies
+      You can set persistence policies for BookKeeper at the namespace level
```

Comment:
-> readme: how to set the persistence policy in line with what we'd like to test (replication, synchronous or asynchronous)